src/components/content.js

```diff
@@ -144,14 +144,35 @@ class Content extends React.Component {
   }
 
   /**
+   * The React ref method to set the root content element locally.
+   *
+   * @param {Element} n
+   */
+
+  ref = (n) => this.element = n
+
+  /**
+   * Check if an `event` is being fired from inside a non-contentediable child
+   * element, in which case we'll want to ignore it.
+   *
+   * @param {Event} event
+   * @return {Boolean}
+   */
+
+  isNonEditable = (event) => {
```

Comment:
This is a slightly bigger ask, but can you change this method to be called something more clear and positive (like `isInContentEditable`) so that the logic isn't double-negative anymore which is hard to read?

This was my fault for setting it up that way to begin with, but as we add more logic it's just getting less understandable haha